gtsfm/loader/argoverse_dataset_loader.py

```diff
@@ -30,15 +30,19 @@ def __init__(
         max_num_imgs: int = 20,
         max_lookahead_sec: float = 2,
         camera_name: str = "ring_front_center",
+        max_resolution: int = 760,
     ) -> None:
         """Select the image paths and their corresponding timestamps for images to feed to GTSFM.
         Args:
             dataset_dir: directory where raw Argoverse logs are stored on disk
             log_id: unique ID of vehicle log
             stride: sampling rate, e.g. every 2 images, every 4 images, etc.
             max_num_imgs: number of images to load (starting from beginning of log sequence)
+            max_resolution: integer representing maximum length of image's short side
+               e.g. for 1080p (1920 x 1080), max_resolution would be 1080
         """
         self._log_id = log_id
+        self._max_resolution = max_resolution
```

Comment:
should be passed to parent constructor, right?

Reply:
Thanks for the review, @ayushbaid. Would the parent constructor be the only place where `max_resolution` be saved as a class attribute? and the only thing we do in that constructor is set that attribute? and not in the child constructor?

Reply:
maybe we should call `super().__init__()` as the very first thing in the child constructor, to set that attribute?

Reply:
`super().__init__(max_resolution)`, right?

This will help fail early instead of the getter function.

Reply:
Cool. I updated it @ayushbaid . Let me know if you like the design better now.